composer/scripts/install-generis.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
be careful with these kind of scripts, always think about taoPremium!

Reply:
Discussed with @siwane, this will probably be done differently